Hazel/src/Platform/OpenGL/OpenGLContext.cpp

```diff
@@ -24,6 +24,14 @@ namespace Hazel {
 		HZ_CORE_INFO("  Renderer: {0}", glGetString(GL_RENDERER));
 		HZ_CORE_INFO("  Version: {0}", glGetString(GL_VERSION));
 
+		#ifdef HZ_ENABLE_ASSERTS
```

Comment:
I would remove the ifdef statement, the optimizer should take care of this as we never use the result?
If someone comes up with a reason to leave it, at least remove one indentation level on the #ifdef and #endif so they are same indented as the `{` and `}`

Reply:
The optimizer won't remove it as it can prove it's pure with no side affects, however it's an insignificant cost at startup so doing it even with asserts disabled should be fine.

Reply:
The ifdef is there, because if asserts are disabled then the whole thing is useless.

Reply:
I know, but it only runs during initialisation/loading of the engine, I imagine this doesn't impact the code alot, like reductor mensioned, But yes, we can leave this indeed. In that case, at least remove one indentation level on the #ifdef and #endif so they are same indented as the `{` and `}`

```cpp
	void OpenGLContext::Init()
	{
		//...

	#ifdef HZ_ENABLE_ASSERTS
		int versionMajor, versionMinor;
		glGetIntegerv(GL_MAJOR_VERSION, &versionMajor);
		glGetIntegerv(GL_MINOR_VERSION, &versionMinor);
		HZ_CORE_ASSERT(versionMajor >= 4 && versionMinor >= 5, "Hazel requires at least OpenGL version 4.5!");
	#endif

Reply:
Fixed the indentation. @LovelySanta